### ISSUE TITLE
Kick command - doesn't give reason and crashes the clent 

### ISSUE THREAD

**Issue by ren-kun** [StarryPy/StarryPy3k#109]
Hello,

based on a few reports - The kick command doesn't show the reason when putting..  /kick player "reason"

It rather crashes the client and throws this error

![image](https://user-images.githubusercontent.com/1111522/27845204-1cff6e56-6170-11e7-84b0-8c637ce48d72.png)

**Comment by medeor413** [comment]
Yeah, this is a known issue that's been around for a while; there are some additional steps to properly kick someone that we haven't quite figured out. We're intending to fix this soon, though.
